codex-ui/dev/Playground.vue

```diff
@@ -1,7 +1,83 @@
 <template>
   <div :class="$style.playground">
     <Heading :level="1">Playground</Heading>
+    <Heading :level="3">Mode:</Heading>
+    <div :class="$style.buttons">
+      <Button
```

Comment:
Maybe use radio buttons instead of regular buttons?

Reply:
We don't have radio components right now. I think, it's ok to use button For the playground at the moment

Reply:
radiobutton-like behavior is implemented incorrectly at the moment. I suggest using native radiobuttons for simplicity. Otherwise there should be a behaviour fix